pkg/middleware/logger.go

```diff
@@ -33,7 +33,7 @@ func Logger(cfg *setting.Cfg) web.Handler {
 		c.Next()
 
 		timeTaken := time.Since(start) / time.Millisecond
-
+		duration := (time.Since(start)).String()
```

Comment:
```suggestion
		duration := time.Since(start).String()
```
I think you should be able to remote the brackets.